pkg/controller/statefulset/stateful_set_control_test.go

```diff
@@ -1993,6 +2100,7 @@ func scaleUpStatefulSetControl(set *apps.StatefulSet,
 		if err := invariants(set, spc); err != nil {
 			return err
 		}
+		//fmt.Printf("Ravig pod conditions %v %v", set.Status.ReadyReplicas, *set.Spec.Replicas)
```

Comment:
This should be removed?

Reply:
Yup, feel free to open a PR and ping me with it :)